docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -30,49 +30,92 @@ tags:
+Your [Usage](https://cloud.temporal.io/usage) provides an overview of your usage across your Namespaces.
+In addition, Account Owners can review [cost breakdowns](#cost-by-namespace) by Namespace.
 
-## How to navigate to your invoices
+## Current balance {#current-balance}
+
+Your current balance card shows the balance for your current billing cycle and the date it was last updated.
+This balance adjusts with use and appears on the first line of your Invoices table.
+
+:::note Billing Cycles
+
+Billing cycles normally begin on the first of the month (UTC).
+The minimum support fee for your first month is prorated based on your sign-up date.
+
+:::
+
+## Recent bill {#recent-bill}
+
+The "Recent Bill" card displays the previous bill amount.
+
+![Recent bill card showing a balance of $0.00](/img/billing/billing-card.png)
+
+* If you are paying invoices through the Stripe portal, you'll also see a **Pay Now** button, which takes you to the Stripe portal to complete your payment.
+* If your account is set up for auto-payment, you don’t need to pay the bill manually, however you can choose to make a manual payment whenever you wish.
+
+## Invoices {#invoice}
 
 To review your invoices, follow these steps:
 
-1. Click **Settings** on your left sidenav.
-2. Select the **Billing** tab.
-3. Under the **Invoices** section, download the invoice(s) you wish to review.
+1. Click **Billing** on your left-side vertical navigation.
+2. Under **Invoices**, click the icon to download the invoice(s) you want to review.
 
-## Current balance {#current-balance}
 
-The current balance card displays the balance accrued thus far in the billing cycle. You will see the date when this data was last updated. Note, this information will change based on usage. This data also corresponds to the first line in the Invoices table. 
+The Invoices table shows the following information:
 
-The typical billing cycle begins on the first of the month (UTC). The minimum support fee for your first month is prorated based on when you sign up.
+- Date (UTC): The date range covered by the invoice.
+- Type: The type of invoice, such as credit purchase or cloud usage.
+- Credit Granted: The total credits added to your account.
+- Credit Purchase Amount: The amount paid for purchasing credits.
+- Credit Usage: The credits used during the billing cycle.
+- Subtotal: The total amount of the invoice before any adjustments.
+- Balance Due: The amount to pay after applying credits.
 
-## Recent bill {#recent-bill}
+![Billing page showing Invoices tab](/img/billing/billing-invoices.png)
+
+:::note Current Month Invoice
+
+During the current billing period, your invoice will not been finalized and the download option will not be available.
+
+:::
+
+## Credits {#credit-table}
+
+The following information appears under the credits table.
+
+- Effective At (UTC): The date when the credit grant became effective
```

Comment:
Lines 81-87 have bullets that end with periods but lines 101-104 don't. Let's try to keep it consistent?

Reply:
Argh! My nemesis. None of the bullet lines should have punctuation. I'll audit the file for all bullets that I accidentally introduced

Reply:
Audited and done. Thank goodness for regex.